### PR TITLE
Add plumbing for AD7 evaluation / property info flags

### PR DESCRIPTION
**Pull request by caslan**
- This change routes incoming evaluation / property flags to the debugger the MI engine is talking to.
- Currently, only clrdbg supports receiving these flags (via the override of VarCreate, VarListChildren) and other debuggers continue to work as before.

### CODE REVIEW

File: src/MICore/MICommandFactory.cs [hunk 1]
```diff
@@ -921,5 +930,21 @@ public override void DecodeExceptionReceivedProperties(Results miExceptionResult
...
+        {
+            string command = string.Format("-var-create - * \"{0}\" --evalFlags {1}", expression, (uint)dwFlags);
+            Results results = await ThreadFrameCmdAsync(command, resultClass, threadId, frameLevel);
+
+            return results;
+        }
+
+        public override async Task<Results> VarListChildren(string variableReference, enum_DEBUGPROP_INFO_FLAGS dwFlags, ResultClass resultClass = ResultClass.done)
+        {
+            string command = string.Format("-var-list-children --simple-values \"{0}\" --propInfoFlags {1}", variableReference, dwFlags);
```
Comment (gregg-miskelly) [inline src/MICore/MICommandFactory.cs]: Do we need a cast of 'dwFlags' to uint?

**Comment by gregg-miskelly** [comment]
Signing off with comments.

**Comment by chuckries** [comment]
:+1:

**Comment by paulmaybee** [comment]
:+1:

**Comment by caslan** [comment]
Merging with Gregg's suggested change.

**Pull request merged**